etc/kayobe/kolla/config/keepalived/keepalived.conf

```diff
@@ -37,6 +37,10 @@ vrrp_instance kolla_internal_vip_{{ keepalived_virtual_router_id }} {
         {{ kolla_external_vip_interface }}
     }
 {% endif %}
+    virtual_routes {
+        128.232.224.0/23 dev {{ kolla_external_vip_interface }} scope link
+        dev {{ kolla_external_vip_interface }} to default via 128.232.225.254
+    }
```

Comment:
ah, its annoying we need a custom one, I guess its probably worth upstreaming this option of extra routes.